packages/babel-core/test/parse.js

```diff
@@ -28,4 +28,32 @@ describe("parse", function() {
     });
     expect(JSON.parse(JSON.stringify(result))).toEqual(output);
   });
+
```

Comment:
parse.js was the easiest place to insert this test. I'm not sure where it should go. My first instinct was to add it to `test/transformation.js`, but there weren't any describe'd tests there...